mrcnn/__init__.py

```diff
@@ -0,0 +1,18 @@
+import os
+import logging
+
+import matplotlib
+import numpy as np
+
+# in case you are running on machine without display, e.g. server
+if os.environ.get('DISPLAY', '') == '' \
+        and matplotlib.rcParams['backend'] != 'agg':
+    logging.warning('No display found. Using non-interactive Agg backend')
+    # https://matplotlib.org/faq/usage_faq.html
+    matplotlib.use('Agg')
```

Comment:
I could be wrong, but I think matplotlib does its best to handle different environments automatically. In which case is this needed? Also, if it is needed then let's have it in `visualize.py` or somewhere else more visible in case someone needs to edit it for their environment. Maybe in a separate PR so we don't cause delay to this one.

Reply:
This is just preventing crash visualization in figures if you run it on a server without any screen or session (also some CI have similar characteristic)